pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[project]
+name = "slider"
+version = "0.8.0"
+description = "Utilities for working with osu! files and data."
+authors = [{name="Joe Jevnik", email="joejev@gmail.com"}]
+license = {text="LGPLv3+"}
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Games/Entertainment",
+]
+requires-python = ">=3.7"
```

Comment:
the readme currently mentions that slider works with 3.6+. I haven't tested this in recent times, but I don't recall changing anything that would increase this. I'm just curious if you tested this yourself, or just chose 3.7 arbitrarily?